spec/features/proposal_spec.rb

```diff
@@ -246,4 +249,31 @@
       expect(page).to have_text(proposal.title)
     end
   end
+
+  context "when declined" do
+    let(:proposal) { create(:proposal) }
+    let(:speaker) { create(:speaker, proposal: proposal, user: user) }
+
+    before do
+      proposal.speakers << speaker
```

Comment:
Good question. I modeled it after the other tests in this spec. Would you recommend putting it inside the before block instead?